### ISSUE TITLE
Fail on init this project via npx and script/setup.sh

### ISSUE THREAD

**Issue by MinhOmega** [haydenbleasel/next-forge#222]
I have read [this issue](https://github.com/haydenbleasel/next-forge/issues/189) but it still happen for me with step bellow

![CleanShot 2024-11-13 at 09 56 50](https://github.com/user-attachments/assets/e86cbb42-47e3-4173-9417-d6cca24b8e78)
When cloning this repo and running `scripts/setup.sh` it still does not work for me.
```
./scripts/setup.sh
Setting up next-forge...
Installing dependencies...
Scope: 2 of 21 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +314
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

   ╭──────────────────────────────────────────────────────────────────╮
   │                                                                  │
   │                Update available! 9.11.0 → 9.12.3.                │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v9.12.3   │
   │                Run "pnpm add -g pnpm" to update.                 │
   │                                                                  │
   │         Follow @pnpmjs for updates: https://x.com/pnpmjs         │
   │                                                                  │
   ╰──────────────────────────────────────────────────────────────────╯

Progress: resolved 314, reused 314, downloaded 0, added 314, done

devDependencies:
+ @auto-it/all-contributors 11.3.0
+ @auto-it/first-time-contributor 11.3.0
+ @biomejs/biome 1.9.4
+ @repo/typescript-config 0.0.0 <- packages/typescript-config
+ turbo 2.2.3
+ typescript 5.6.3
+ ultracite 4.1.6

Done in 3.3s
Copying .env.example files to .env.local...
Deleting internal content...
rm: .github/FUNDING.md: No such file or directory
Setting up Prisma...

> next-forge@2.13.7 build /Users/minhvo/Desktop/Web/next-forge
> turbo build "--filter" "@repo/database"

turbo 2.2.3

• Packages in scope: @repo/database
• Running build in 1 packages
• Remote caching disabled
@repo/database:build: cache miss, executing de045ea0d8c1aa8c
@repo/database:build:
@repo/database:build:
@repo/database:build: > @repo/database@0.0.0 build /Users/minhvo/Desktop/Web/next-forge/packages/database
@repo/database:build: > prisma generate --no-hints
@repo/database:build:
@repo/database:build: sh: prisma: command not found
@repo/database:build:  ELIFECYCLE  Command failed.
@repo/database:build:  WARN   Local package.json exists, but node_modules missing, did you mean to install?
@repo/database:build: ERROR: command finished with error: command (/Users/minhvo/Desktop/Web/next-forge/packages/database) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run build exited (1)
@repo/database#build: command (/Users/minhvo/Desktop/Web/next-forge/packages/database) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run build exited (1)

 Tasks:    0 successful, 1 total
Cached:    0 cached, 1 total
  Time:    762ms
Failed:    @repo/database#build

 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed with exit code 1.
Setup complete! Deleting scripts folder...
```
Please take a look, here is my info:
- MacBook Pro M1 pro - MacOS 15.1
- NodeJS 20.16
- Pnpm 9.11.0
- npm 10.8.1

**Comment by trung1051996** [comment]
I facing this issue when cloning.
NodeJS 18.19.0
Pnpm 9.12.2
npm 10.2.3
![Screenshot 2024-11-13 at 14 50 51](https://github.com/user-attachments/assets/478fa996-fea4-4cc8-8998-216553fffb62)

**Comment by haydenbleasel** [comment]
@MinhOmega I'm seeing 3 potential issues here:

1. The GitHub URL not working. Seems to be reported here: https://github.com/vercel/next.js/issues/19277. I can't repro this but i'll do more digging.
2. `.github/funding.yml` not found. I also couldn't repro this on a fresh `init`. Need some help repro'ing this.
3. `prisma: command not found` means your node dependencies aren't installed.

I'll spin up a repl.it or something to attempt to repro these in a sandbox environment.

**Comment by haydenbleasel** [comment]
From the issue linked, looks like wrapping the `example` url in double quotes helped. I'll give that a try.

**Comment by MinhOmega** [comment]
Yes, I think there might be an issue with my Mac or the environment. Even though the initialization was successful, the command `pnpm install` did not install all `node_modules` for all packages. So, when I run `pnpm dev`, it throws an error. If you've encountered this issue before, I would appreciate your help. 

Thank you!
```
pnpm dev

> next-forge@2.13.9 dev /Users/minhvo/Desktop/Web/test
> turbo dev

turbo 2.2.3

• Packages in scope: @repo/analytics, @repo/auth, @repo/database, @repo/design-system, @repo/email, @repo/env, @repo/feature-flags, @repo/next-config, @repo/observability, @repo/payments, @repo/security, @repo/seo, @repo/tailwind-config, @repo/typescript-config, api, app, docs, email, studio, web
• Running dev in 20 packages
• Remote caching disabled
app:dev: cache bypass, force executing a9d44377e11b4b29
email:dev: cache bypass, force executing 95c0e6b496739a05
docs:dev: cache bypass, force executing 91cb8455b2773cfb
api:dev: cache bypass, force executing 24f3c541ddd20714
web:dev: cache bypass, force executing bd54f8fa1ba43208
studio:dev: cache bypass, force executing 52defa4733a37421
email:dev:
email:dev: > email@0.0.0 dev /Users/minhvo/Desktop/Web/test/apps/email
email:dev: > email dev --port 3003
email:dev:
app:dev:
app:dev: > app@ dev /Users/minhvo/Desktop/Web/test/apps/app
app:dev: > next dev -p 3000
app:dev:
email:dev: sh: email: command not found
email:dev:  ELIFECYCLE  Command failed.
email:dev:  WARN   Local package.json exists, but node_modules missing, did you mean to install?
app:dev: sh: next: command not found
app:dev:  ELIFECYCLE  Command failed.
app:dev:  WARN   Local package.json exists, but node_modules missing, did you mean to install?
web:dev:
web:dev: > web@ dev /Users/minhvo/Desktop/Web/test/apps/web
web:dev: > next dev -p 3001
web:dev:
studio:dev:
studio:dev: > studio@0.0.0 dev /Users/minhvo/Desktop/Web/test/apps/studio
studio:dev: > prisma studio --schema ../../packages/database/prisma/schema.prisma --port 3005
studio:dev:
web:dev: sh: next: command not found
api:dev:
api:dev: > api@ dev /Users/minhvo/Desktop/Web/test/apps/api
api:dev: > concurrently "pnpm:next" "pnpm:stripe"
api:dev:
web:dev:  ELIFECYCLE  Command failed.
web:dev:  WARN   Local package.json exists, but node_modules missing, did you mean to install?
email:dev: ERROR: command finished with error: command (/Users/minhvo/Desktop/Web/test/apps/email) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run dev exited (1)
docs:dev:
email#dev: command (/Users/minhvo/Desktop/Web/test/apps/email) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run dev exited (1)

 Tasks:    0 successful, 6 total
Cached:    0 cached, 6 total
  Time:    684ms
Failed:    email#dev

 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed with exit code 1.
 ```

**Comment by haydenbleasel** [comment]
repl.it log:

```
npx next-forge init test-11
Need to install the following packages:
next-forge@2.13.9
Ok to proceed? (y) 

.../19324989595-a6                       | Progress: resolved 1, .../19324989595-a6                       |   +1 +
.../19324989595-a6                       | Progress: resolved 1, .../19324989595-a6                       | Progress: resolved 1, .../19324989595-a6                       | Progress: resolved 1, reused 0, downloaded 1, added 1, done
Creating a new Next.js app in /home/runner/next-forge-init/test-11.

Downloading files from repo https://github.com/haydenbleasel/next-forge. This might take a moment.

Installing packages. This might take a couple of minutes.

Scope: all 21 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +1437
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

╭───────────────────────────────────────────────────────────────╮│                                                               ││               Update available! 9.9.0 → 9.12.3.               ││                          Changelog:                           ││       https://github.com/pnpm/pnpm/releases/tag/v9.12.3       ││               Run "pnpm add -g pnpm" to update.               ││                                                               ││       Follow @pnpmjs for updates: https://x.com/pnpmjs        ││                                                               │╰───────────────────────────────────────────────────────────────╯

Downloading @swc/core-linux-x64-gnu@1.3.101: 15.81 MB/15.81 MB, donenloading @swc/core-linux-x64-gnu@1.3.101: 3.29 kB/15.81 MB
Downloading @swc/core-linux-x64-musl@1.3.101: 18.85 MB/18.85 MB, done
Downloading turbo-linux-64@2.2.3: 13.29 MB/13.29 MB, done
Downloading @biomejs/cli-linux-x64-musl@1.9.4: 10.72 MB/10.72 MB, done
Downloading @biomejs/cli-linux-x64@1.9.4: 10.86 MB/10.86 MB, done
Downloading next@15.0.3: 24.75 MB/24.75 MB, done
Downloading @img/sharp-libvips-linux-x64@1.0.4: 7.06 MB/7.06 MB, done
Downloading @img/sharp-libvips-linuxmusl-x64@1.0.4: 7.20 MB/7.20 MB, done
Downloading @next/swc-linux-x64-musl@15.0.3: 55.51 MB/55.51 MB, done
Downloading @next/swc-linux-x64-gnu@15.0.3: 45.67 MB/45.67 MB, done
Downloading @sentry/cli-linux-x64@2.37.0: 5.96 MB/5.96 MB, done
Downloading react-email@3.0.2: 16.02 MB/16.02 MB, done
Downloading next@14.2.10: 20.66 MB/20.66 MB, done
Downloading @next/swc-linux-x64-musl@14.2.10: 50.27 MB/50.27 MB, done
Downloading @next/swc-linux-x64-gnu@14.2.10: 41.91 MB/41.91 MB, done
Downloading sqip-plugin-primitive@1.0.0-alpha.53: 8.26 MB/8.26 MB, done
Downloading next@15.0.1: 22.12 MB/22.12 MB, done
Downloading @next/swc-linux-x64-gnu@15.0.1: 45.66 MB/45.66 MB, done
Downloading @next/swc-linux-x64-musl@15.0.1: 55.70 MB/55.70 MB, done
 WARN  GET https://registry.npmjs.org/@types/lodash/-/lodash-4.17.4.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/next/-/next-15.0.1.tgz error (ERR_PNPM_UNKNOWN). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/@octokit/plugin-paginate-rest/-/plugin-paginate-rest-2.21.3.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/@octokit/endpoint/-/endpoint-6.0.12.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/onetime/-/onetime-5.1.2.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 ERR_PNPM_ENOENT  ENOENT: no such file or directory, mkdir '/home/runner/next-forge-init/test-11/node_modules/.pnpm/qs@6.12.1/node_modules/qs_tmp_195/dist'
Progress: resolved 1437, reused 0, downloaded 1340, added 1336

Aborting installation.
  pnpm install has failed.

Setting up next-forge...
Installing dependencies...
Scope: all 21 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +1437
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 WARN  GET https://registry.npmjs.org/onetime/-/onetime-5.1.2.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/import-cwd/-/import-cwd-3.0.0.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/@octokit/endpoint/-/endpoint-6.0.12.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/@octokit/plugin-paginate-rest/-/plugin-paginate-rest-2.21.3.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/ts-node/-/ts-node-9.1.1.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 WARN  GET https://registry.npmjs.org/parse-json/-/parse-json-4.0.0.tgz error (ERR_PNPM_Unknown system error -122). Will retry in 10 seconds. 2 retries left.
 ERR_PNPM_Unknown system error -122  Unknown system error -122: Unknown system error -122, copyfile '/home/runner/next-forge-init/.local/share/pnpm/store/v3/files/20/e2ce2b6f7a593b431ff57bfba190dfc6b759eb9bfc6358b979c0f2f799dafe4eb6a175f5fa07349f46ee3f0970681a5106c31af28ae29227137177309ed808' -> '/home/runner/next-forge-init/test-11/node_modules/.pnpm/objectorarray@1.0.5/node_modules/objectorarray_tmp_219/LICENSE'
Progress: resolved 1437, reused 1340, downloaded 0, added 0
Copying .env.example files to .env.local...
Deleting internal content...
rm: cannot remove '.github/FUNDING.md': No such file or directory
Setting up Prisma...

> next-forge@2.13.9 build /home/runner/next-forge-init/test-11
> turbo build "--filter" "@repo/database"

sh: line 1: turbo: command not found
 ELIFECYCLE  Command failed.
Setup complete! Deleting scripts folder...
```

**Comment by haydenbleasel** [comment]
Whoops, funding is a yml file, not Markdown. That should be an easy fix.

**Comment by haydenbleasel** [comment]
@MinhOmega Glad it's working for you now!

@trung1051996 Would you be able to re-init and see if it's still an issue?

**Comment by MinhOmega** [comment]
Many thanks, @haydenbleasel ! However, I'm still facing an issue with installing `pnpm i` in the monorepo; it still shows `Scope: 2 of 21 workspace projects` 🥲🥲🥲.
```
npx next-forge@latest init nineten
Need to install the following packages:
next-forge@2.13.12
Ok to proceed? (y) y

.../1932689353c-869e                     |   +1 +
.../1932689353c-869e                     | Progress: resolved 1, reused 0, downloaded 1, added 1, done
Creating a new Next.js app in /Users/minhvo/Desktop/Web/nineten.

Downloading files from repo https://github.com/haydenbleasel/next-forge. This might take a moment.

Installing packages. This might take a couple of minutes.

Scope: 2 of 21 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +314
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

   ╭──────────────────────────────────────────────────────────────────╮
   │                                                                  │
   │                Update available! 9.12.3 → 9.13.0.                │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v9.13.0   │
   │                Run "pnpm self-update" to update.                 │
   │                                                                  │
   │         Follow @pnpmjs for updates: https://x.com/pnpmjs         │
   │                                                                  │
   ╰──────────────────────────────────────────────────────────────────╯

Progress: resolved 314, reused 314, downloaded 0, added 314, done

devDependencies:
+ @auto-it/all-contributors 11.3.0
+ @auto-it/first-time-contributor 11.3.0
+ @biomejs/biome 1.9.4
+ @repo/typescript-config 0.0.0 <- packages/typescript-config
+ turbo 2.2.3
+ typescript 5.6.3
+ ultracite 4.1.6

Done in 3.4s

Initialized a git repository.

Success! Created nineten at /Users/minhvo/Desktop/Web/nineten
Inside that directory, you can run several commands:

  pnpm run dev
    Starts the development server.

  pnpm run build
    Builds the app for production.

  pnpm start
    Runs the built app in production mode.

We suggest that you begin by typing:

  cd nineten
  pnpm run dev

Setting up next-forge...
Installing dependencies...
Scope: 2 of 21 workspace projects
Lockfile is up to date, resolution step is skipped
Already up to date

   ╭──────────────────────────────────────────────────────────────────╮
   │                                                                  │
   │                Update available! 9.12.3 → 9.13.0.                │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v9.13.0   │
   │                Run "pnpm self-update" to update.                 │
   │                                                                  │
   │         Follow @pnpmjs for updates: https://x.com/pnpmjs         │
   │                                                                  │
   ╰──────────────────────────────────────────────────────────────────╯

Done in 902ms
Copying .env.example files to .env.local...
Deleting internal content...
Setting up Prisma...

> next-forge@2.13.12 build /Users/minhvo/Desktop/Web/nineten
> turbo build "--filter" "@repo/database"

turbo 2.2.3

• Packages in scope: @repo/database
• Running build in 1 packages
• Remote caching disabled
@repo/database:build: cache miss, executing de045ea0d8c1aa8c
@repo/database:build:
@repo/database:build:
@repo/database:build: > @repo/database@0.0.0 build /Users/minhvo/Desktop/Web/nineten/packages/database
@repo/database:build: > prisma generate --no-hints
@repo/database:build:
@repo/database:build: sh: prisma: command not found
@repo/database:build:  ELIFECYCLE  Command failed.
@repo/database:build:  WARN   Local package.json exists, but node_modules missing, did you mean to install?
@repo/database:build: ERROR: command finished with error: command (/Users/minhvo/Desktop/Web/nineten/packages/database) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run build exited (1)
@repo/database#build: command (/Users/minhvo/Desktop/Web/nineten/packages/database) /Users/minhvo/.nvm/versions/node/v20.16.0/bin/pnpm run build exited (1)

 Tasks:    0 successful, 1 total
Cached:    0 cached, 1 total
  Time:    1.236s
Failed:    @repo/database#build

 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed with exit code 1.
Setup complete! Deleting scripts folder...
```
I think we can close this issue for now, and I will try to fix it on my machine.

**Comment by haydenbleasel** [comment]
Okay interesting, seems like `pnpm install` is only installing 2 of the 21 packages and apps. Not sure why this would happen but I'll do some research.
